lib/segment/src/index/field_index/geo_index.rs

```diff
@@ -1 +1,531 @@
+use crate::types::{GeoBoundingBox, GeoPoint, GeoRadius};
+use geo::Coordinate;
+use geohash::{decode, encode, neighbor, Direction};
+use std::ops::Range;
 
+type GeoHash = String;
```

Comment:
```suggestion
struct GeoHash(String);
```
Wrapping it this way will help with type checking and maybe prevent a few bugs, as you will need to explicitly convert/wrap string into GeoHash

Reply:
Later I checked the full logic and seems that yes, it will not help much for now.